liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/UndeployAppMojo.java

```diff
@@ -136,8 +136,7 @@ protected void undeployApp(File file) throws MojoExecutionException {
                 Map<String, File> libertyDirPropertyFiles = getLibertyDirectoryPropertyFiles();
                 CommonLogger logger = new CommonLogger(getLog());
                 setLog(logger.getLog());
-                getServerConfigDocument(logger, serverXML, configDirectory,
-                bootstrapPropertiesFile, combinedBootstrapProperties, serverEnvFile, false, libertyDirPropertyFiles);
+                scd = getServerConfigDocument(logger, libertyDirPropertyFiles);
```

Comment:
Do not need to get serverXML file at line 134 anymore.

Reply:
Done